### ISSUE TITLE
make Chapeldown Manor playable with Base-set only

### ISSUE THREAD

**Issue by antontimmermans** [antontimmermans/MOMCDM#2]
This discussion threat is moved from the Valkyrie repository to MOMCDM.

### ***** SPOILER ALERT*****

@scrubbless @chrismaliszewski There might be a possibility to make it playable with the base set only.
This will then result in that only 1 out of 2 mansions will be selected but all 5 stories will then still be possible. The Mansion that is easiest to do is then the one that starts with the Lobby from the base set (with the red carpet).

I can replace the Elli Horowitz token with the Purple Lady Token. That is easy because no scenario clues are related to this person token. All other Person Tokens are from the base set, so that is OK.

Then I might need to replace any SoA tile used in that mansion with similar rooms from the base set. However, their rough description and function must match. (type of room, type of floor, nr of chairs and their color)
For example: replacing the ResearchLibrary: it must be a room with a red chair, have a wooden floor and must fit in the description of a 'book-filled' room. The Library might be re-used for this but i would have to double check all the details of the clues. I hope to prevent rewriting the clues, because that cause me a lot of headache.

So in short.. I see possibilities indeed, i'll look into it the next weeks.

**Comment by scrubbless** [comment]
That's great! 
In a stress and strain I had thrall monsters in a few places, when I removed btt I added the test #btt ==1 to the spawn the. Created an alternative spawn (deepone hybrid in this case) to act as that monster. Fortunately they were custom monsters so I could just use the same activation, evade and horror events.

I then did the same as you are suggesting, swapped out certain tiles with base/RN tiles where possible and tweaked the text when going into the room.

**Comment by antontimmermans** [comment]
I can find a proper replacement for the Library, but the chapel is going to be a problem. The only option I see that matches clues, is to turn the chapel into a diningroom+kitchen. This does make it a bit akward to pretend that it used to be a chapel but it may work .
(not gonna change the scenario name in _Diningdown Manor_")

**Comment by scrubbless** [comment]
Haha Diningdown manor has a certain ring to it (not a good one).

I personally would not change the core tile in your scenario. I couldn't do a stress and strain without a lab or zombies so RN had to stay.

**Comment by chrismaliszewski** [comment]
Diningdown Manor doesn't seem right :) What about Belltowerdown or... Atticdown Manor? Interestingly enough, https://www.google.com/search?q=Attic+Down&tbm=isch, the latter could be a good motive for a scenario; I mean, hidden stairs to the attic where all the bad things happened. Just saying...

I didn't have time to check the scenario again, sorry! I have so many things to do and finish at the moment...

Just so you know, I was just hoping for the possibility but I also strongly agree with @scrubbless, you shouldn't change the core idea especially if the scenario may lose something or it is not possible. If so, let it stay as it is :).

**Comment by zcaalock** [comment]
Use the chapel from 1st edition, lot of people has it

**Comment by scrubbless** [comment]
The problem is, you either make SoA mandatory or you make RN mandatory. There is no way to make SoA **or** RN mandatory.

Ideally if you want to replace a core tile, it should be with a base tile, so that you can add optional variations that use the tiles based on what expansions players have.

In this case, I would not change the scenario dramatically to swap an SoA mandatory expansion to a RN mandatory expansion. SoA is the most popular expansion, 1st edition (RN) is also quite popular due to many players having 1st Ed, but the tile is quite different so may cause problems swapping out.

**Comment by antontimmermans** [comment]
@chrismaliszewski, @zcaalock, @scrubbles. Thank you for your ideas and considerations on this subject.

I managed to replace tiles, person tokens and items from SoA with Base set equivalents.

If SoA is not available, one the mansion's layout will be slightly modified with the alternative tiles and the other layout is not available. This will reduce the random story/mansion combination by half.

It needs a bit more testing (and 1 polish text translation) before i dare to release it. any volunteers? :)

**Comment by chrismaliszewski** [comment]
Sure, no problem. I'm still away so I won't be able to test the scenario yet with the proper game board but I surely can translate the scenario :).

**Comment by scrubbless** [comment]
Awesome @antontimmermans

As long as people know adding SoA adds huge variety then it's great.

**Comment by antontimmermans** [comment]
@chrismaliszewski ,
This test version is playable without SoA and is translated by Zcaalock in Polish
[ChapelDownManorv1.3.zip](https://github.com/antontimmermans/MOMCDM/files/3057541/ChapelDownManorv1.3.zip)

**Comment by chrismaliszewski** [comment]
As for now, I noticed a few problems. 
1. I don't exactly remember statuses that a player can have in the base game but I don't remember **Righteous** be on the list. (as I said, I'm still away and don't have the game with me). Check Elli's dialogue and others. 
2. (Replicable) Start the game, open the door on the bottom right, change Phase, start talking with Edna (restart the game until Edna is there), choose the top dialogue option, choose "May I see those papers?" - **nothing happens**! To be able to overcome this issue, you must choose another dialogue option and come back to the one. There is something wrong there.
3. I also have another issue twice. I had a character token displayed in the top left corner and I wasn't able to do anything. Even Main Menu button wasn't displayed. It happened after I started the ritual in the Dining Room. 
4. Check the save: (https://github.com/antontimmermans/MOMCDM/files/3058066/saveAuto.zip)[saveAuto.zip]
There are **many** things wrong with the save game; can't say if it's just a single-time event or something bigger coded in the scenario. 
A. I talked with Ellie, change a Phase and started another turn. I can't click anything on the board. 
B. Change phase to the next one. During the next investigator phase, there is an infinite loop of fuse-box failure. It's possible to click Cancel but see for yourself. 
C. After clicking Cancel, you're moved to Howard but you and "Nevermind" dialogue option does nothing. 

I'll check @zcaalock's translation soon and add my two cents where appropriate.

**Comment by antontimmermans** [comment]
> As for now, I noticed a few problems.
> 
> 1. I don't exactly remember statuses that a player can have in the base game but I don't remember **Righteous** be on the list. (as I said, I'm still away and don't have the game with me). Check Elli's dialogue and others.

True, this needs to be fixed.. i think i will change to focussed.

> 2. (Replicable) Start the game, open the door on the bottom right, change Phase, start talking with Edna (restart the game until Edna is there), choose the top dialogue option, choose "May I see those papers?" - **nothing happens**! To be able to overcome this issue, you must choose another dialogue option and come back to the one. There is something wrong there.

Well spotted. The 3 question dialoge, and 2 question dialogue didn't have the alternative answer yet. in the Soa included version this is not an issue.

> 3. I also have another issue twice. I had a character token displayed in the top left corner and I wasn't able to do anything. Even Main Menu button wasn't displayed. It happened after I started the ritual in the Dining Room.

The character token in the topleft corner must have been Edna and was a side effect from the previous issue.

> 4. Check the save: ([https://github.com/antontimmermans/MOMCDM/files/3058066/saveAuto.zip)[saveAuto.zip]](https://github.com/antontimmermans/MOMCDM/files/3058066/saveAuto.zip)%5BsaveAuto.zip%5D)
>    There are **many** things wrong with the save game; can't say if it's just a single-time event or something bigger coded in the scenario.
>    A. I talked with Ellie, change a Phase and started another turn. I can't click anything on the board.
>    B. Change phase to the next one. During the next investigator phase, there is an infinite loop of fuse-box failure. It's possible to click Cancel but see for yourself.
>    C. After clicking Cancel, you're moved to Howard but you and "Nevermind" dialogue option does nothing.
> 
> I'll check @zcaalock's translation soon and add my two cents where appropriate.

Going through the different turns does indeed give very weird behavior. I don't really understand why (yet). There is nothing fundamentally different between the two versions (includig or excluding SOA) when it comes to going through the turns.  Maybe there is another SoA dependency I am not aware off. This is really unexpected.
None of this weird behavior is happening when SoA is available. I need to spend some more time figuring out where this comes from.

thx for testing sofar, I guess the scenario isn't ready to be played without SoA After all.

FYI: I have a hidden test button in the FIRST Introductionscreen. There is a very small invisible button positioned jsut under the left tip of the text top bar.

**Comment by chrismaliszewski** [comment]
@antontimmermans, I created some notes regarding the English version here https://github.com/antontimmermans/MOMCDM/issues/4#issuecomment-482577868. I checked the Polish version too.

**Comment by chrismaliszewski** [comment]
@antontimmermans, when you create a new version for just the base, let us know and I'll check it again.

**Comment by antontimmermans** [comment]
I cannot find a way to implement this without compromising the story. This enhancement will not be done.
